package.json

```diff
@@ -37,9 +37,6 @@
         "test:watch": "jest --watch",
         "test": "jest"
     },
-    "resolutions": {
```

Comment:
if I"m reading things right here. https://github.com/axios/axios/pull/5530 we could also jump to version 1.2.0 and should be fine.